index.d.ts

```diff
@@ -29,6 +29,47 @@ export namespace Memgraph {
     export function Connect_1(params: any): Promise<Connection>;
     export { Connect_1 as Connect };
 }
+/**
+  * Create Memgraph compatible date object.
+  * @param {number} days - The number of days since 1970-01-01 (unix epoch).
+  */
+export function createMgDate(days: number): {
+    objectType: string;
+    days: number;
+};
+/**
+  * Create Memgraph compatible local time object.
+  *
+  * @param {number} nanoseconds - The number of nanoseconds since midnight.
+  */
+export function createMgLocalTime(nanoseconds: number): {
+    objectType: string;
+    nanoseconds: number;
+};
+/**
+  * Create Memgraph compatible local date time object.
+  * @param {number} seconds - The number of seconds since 1970-01-01 (unix epoch).
```

Comment:
maybe add the `h:m:s` as well ?

Reply:
This is auto-generated, I'll fix it in the `index.js`